src/fight.cpp

```diff
@@ -1058,6 +1058,17 @@ void die(struct char_data * ch)
   raw_kill(ch);
 }
 
+ACMD(do_dw_retrieve)
+{
+  // If they're ready to be docwagon'd out, save them.
+  if (PLR_FLAGGED(ch, PLR_DOCWAGON_READY)) {
```

Comment:
There should probably be a check in here to see if they're wearing a valid modulator at all. Help won't arrive if they're not wearing one.

Reply:
Also, maybe give them an error if they're not mortally wounded? `FAILURE_CASE(GET_POS(ch) != POS_MORTALLYW, "message");` sort of thing

Reply:
Ah, right.